admin_guide/topics/pod-priority-examples.adoc

```diff
@@ -0,0 +1,56 @@
+////
+Pod Preemption Examples
```

Comment:
I'd move the special cases in to the main concept and keep the two examples separate.